README.md

```diff
@@ -178,116 +176,6 @@ func NewFoo() *Foo {
 func (f *|
 ```
 
-### `g:deoplete#sources#go#use_cache` `g:deoplete#sources#go#json_directory`
-#### Static json caching
-
-`g:deoplete#sources#go#use_cache`
-
-| **Default**  | `0`           |
-|--------------|---------------|
-| **Required** | **Recommend** |
-| **Type**     | int           |
-| **Example**  | `1`           |
-
-`g:deoplete#sources#go#json_directory`
-
-| **Default**  | `''`                                   |
-|--------------|----------------------------------------|
-| **Required** | **Recommend**                          |
-| **Type**     | string                                 |
-| **Example**  | `'~/.cache/deoplete/go/$GOOS_$GOARCH'` |
-
-Use static json caching Go stdlib package API.  
-If matched name of stdlib and input package name, it returns the static json data without the use of gocode.  
-and, Possible to get package API if have not `import` of current buffer.
-
-Terms:
-
-- Hook the insert of dot `.` (e.g. `fmt.|`)
-- You typed package name have not `import` current buffer
-- Match the typed package name and json file name
-
-`deoplete-go` will parse `g:deoplete#sources#go#json_directory` directory. You can define of json data directory.  
-Recommend is `~/.cache/deoplete/go/$GOOS_$GOARCH` because this directly use other deoplete sources(e.g. deoplete-deji).
-
-Also, See [How to use static json caching](#how-to-use-static-json-caching)
-
-```vim
-let g:deoplete#sources#go#use_cache = 1
-let g:deoplete#sources#go#json_directory = '/path/to/data_dir'
-```
-
-### `g:deoplete#sources#go#cgo`
```

Comment:
@Shougo It seems to work successfully. But why also delete `g:deoplete#sources#go#cgo` document?

Reply:
Oh, it is my fault.